dist/index.html

```diff
@@ -121,6 +122,33 @@ <h2>Total Cost</h2>
       </section>
       <section class="recipe hidden" id="recipe">
       </section>
+    </section>
+    <section class="pantry hidden" id="pantry-view">
+      <section class="pantryContents">
+        <div class="dropDropMenu">
```

Comment:
you dont need to incorporate this, this can be a note during the accessibility audit: we could rename this "div" to "menu".